File: Daily-Challenges/1344_clock_angle.java
class Solution {
    public double angleClock(int hour, int minutes) {
        
        double hand1 = hour;
        if (hour == 12) hand1 = 0;
        hand1 += minutes/60.0;

        double hand2 = minutes/5.0;
        double angle = Math.abs(hand1 - hand2) * 30;

        if (angle <= 180) return angle;
        return 360 - angle;
    }
}

File: Daily-Challenges/1344_clock_angle.py
class Solution:
    def angleClock(self, hour: int, minutes: int) -> float:
        hand1 = hour if hour!=12 else 0
        hand1 += minutes/60
        hand2 = minutes/5
        diff = abs(hand1 - hand2)
        angle = diff*30
        return angle if angle <= 180 else 360 - angle